qt/scientific_interfaces/Direct/ALFView_model.cpp

```diff
@@ -0,0 +1,124 @@
+#include "ALFView_model.h"
```

Comment:
This should be below the license agreement or it will break the script we have that updates it.